protocol/v2/qbft/spectest/create_msg_type.go

```diff
@@ -14,7 +14,25 @@ import (
 	spectestingutils "github.com/ssvlabs/ssv-spec/types/testingutils"
 )
 
-func RunCreateMsg(t *testing.T, test *spectests.CreateMsgSpecTest) {
+type CreateMsgSpecTest struct {
+	Name string
+	// ISSUE 217: rename to root
+	Value [32]byte
+	// ISSUE 217: rename to value
+	StateValue                                       []byte
+	Round                                            qbft.Round
+	RoundChangeJustifications, PrepareJustifications []*spectypes.SignedSSVMessage
+	CreateType                                       string
+	ExpectedRoot                                     string
+	ExpectedState                                    spectypes.Root `json:"-"` // Field is ignored by encoding/json"
+	ExpectedError                                    string
+}
+
+func (test *CreateMsgSpecTest) TestName() string {
+	return "qbft create message " + test.Name
+}
+
+func (test *CreateMsgSpecTest) RunCreateMsg(t *testing.T) {
```

Comment:
did you debug and check that all `create` functions run ssv code?

Reply:
I see you are still calling spec logic underneath
https://github.com/bloxapp/ssv/blob/005616572e8845320b43195d9112b4e99abdb379/protocol/v2/qbft/spectest/create_msg_type.go#L96

Reply:
@GalRogozinski, I don't get it what for do we need to copy this to implementation right now. By doing this we just increase amount of copy paste code which has to be maintained in 2 repos. This significantly increases a chance to miss something and create a bug, increases amount of work in future and brings no profit.

Here I don't see any business logic we hide when constructing the qbft messages. We use:
- `CreateProposal`
- `CreatePrepare`
- `CreateCommit` 
just to construct a message and sign it. Thats it.

@MatusKysel **FYI**

Reply:
https://github.com/bloxapp/ssv/blob/7fe53a03491b405d7d2b54f76f58274c645d4125/protocol/v2/qbft/instance/proposal.go#L274-L315
it seems that you have CreateProposal and other Create functions in ssv already...
so by calling the spec impls you are bypassing ssv which isn't good please change